### PR TITLE
Fix ROS Humble build

### PR DESCRIPTION
**Pull request by hello-atharva**
## Summary
Bumps several ROS repos to Humble. stretch_moveit_plugins is now using the latest MoveIt API for Humble. This PR also fixes librealsense installation.

### CODE REVIEW

**Comment by hello-binit** [comment]
Hi Atharva, it looks like your accounts got mixed up. The PR is opened by @hello-atharva, but the commit is authored by @atharva-18 . Can you amend the author on that commit and force push it?

File: factory/22.04/stretch_ros2_humble.repos [hunk 1]
```diff
@@ -6,11 +6,11 @@ repositories:
   moveit_visual_tools:
     type: git
     url: https://github.com/ros-planning/moveit_visual_tools
-    version: 4.0.0
+    version: 4.1.0
   moveit_task_constructor:
```
Comment (hello-chintan) [inline factory/22.04/stretch_ros2_humble.repos:10]: @hello-atharva Binaries for this package are now available. Do we need to source build?

File: factory/22.04/stretch_ros2_humble.repos [hunk 2]
```diff
@@ -21,8 +21,8 @@ repositories:
     version: ros2
   stretch_moveit_plugins:
```
Comment (hello-chintan) [inline factory/22.04/stretch_ros2_humble.repos:22]: @hello-atharva stretch_moveit_plugins will likely be housed in stretch_moveit2 as part of the stretch_ros2 repo as we want all moveit related packages to exist together.

File: factory/22.04/stretch_ros2_humble.repos [hunk 3]
```diff
@@ -6,11 +6,11 @@ repositories:
   moveit_visual_tools:
```
Comment (hello-chintan) [inline factory/22.04/stretch_ros2_humble.repos:6]: @hello-atharva Binaries available. Do we need to source build?

File: factory/22.04/stretch_install_system.sh [hunk 4]
```diff
@@ -67,7 +67,7 @@ echo "###########################################"
 
 echo "Ensuring Ubuntu Universe repository is enabled"
 sudo apt install software-properties-common
-sudo add-apt-repository universe
+sudo add-apt-repository universe # TODO(atharva-18): Do we really need this?
```
Comment (hello-chintan) [inline factory/22.04/stretch_install_system.sh:70]: @hello-atharva Good catch! Feel free to remove it.

**Comment by hello-atharva** [comment]
Looks like latest build using `stretch_ros2` on `rolling` is failing.

**Comment by hello-atharva** [comment]
Are we using the latest API here?
[https://github.com/hello-robot/stretch_ros2/blob/28b6a98bb718260c79cbb65fd810480f18c4865a/stretch_moveit2/stretch_moveit_config/src/moveit_draw.cpp#L143](https://github.com/hello-robot/stretch_ros2/blob/28b6a98bb718260c79cbb65fd810480f18c4865a/stretch_moveit2/stretch_moveit_config/src/moveit_draw.cpp#L143)

It should be instead:

```
if(i <= 1){
      disp_plan.trajectory_ = my_plan.trajectory_;
}
```

**Comment by hello-chintan** [comment]
@hello-atharva You're right. The MoveIt cpp code will be deprecated in Humble as we move to the Python API, so it's not necessary to update it.

**Comment by hello-binit** [comment]
Pending Chintan's notes, I think this looks ready to merge.

File: factory/22.04/stretch_ros2_humble.repos (hunk 1 above)
Comment (hello-atharva) [reply factory/22.04/stretch_ros2_humble.repos:10]: Looks like it! Should we install it here or add that package as a rosdep dependency in stretch_ros2?

File: factory/22.04/stretch_ros2_humble.repos (hunk 2 above)
Comment (hello-atharva) [reply factory/22.04/stretch_ros2_humble.repos:22]: Should we keep it for the time being till we get that or it's safe to remove it?

File: factory/22.04/stretch_ros2_humble.repos (hunk 3 above)
Comment (hello-atharva) [reply factory/22.04/stretch_ros2_humble.repos:6]: Ref: [https://github.com/hello-robot/stretch_install/pull/39#discussion_r1207148207](https://github.com/hello-robot/stretch_install/pull/39#discussion_r1207148207)

File: factory/22.04/stretch_ros2_humble.repos (hunk 2 above)
Comment (hello-chintan) [reply factory/22.04/stretch_ros2_humble.repos:22]: Let's keep it until we add it to stretch_moveit2.

File: factory/22.04/stretch_ros2_humble.repos (hunk 1 above)
Comment (hello-chintan) [reply factory/22.04/stretch_ros2_humble.repos:10]: The latter sounds like a better idea. Could you add it?

**Comment by hello-chintan** [comment]
Looks good. Merging this in.

**Pull request merged**